sgt-back-end/index.js

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable no-console */
+const express = require('express');
+const app = express();
+const pg = require('pg');
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const middleware = express.json();
+app.use('/api/grades', middleware);
```

Comment:
When applying the json middleware we want to apply it to all paths not just the specific path of `/api/grades`. This should be:
```JAVASCRIPT
app.use(middleware);
```